.gitignore

```diff
@@ -1,8 +1,11 @@
 /build/
 /l10n/.transifexrc
+.php_cs.cache
+
 # Composer
 composer.phar
 vendor/
+vendor-bin/owncloud-codestyle/composer.lock
```

Comment:
can be written as `vendor-bin/**/composer.lock` to prevent adding lock files by accident

Reply:
done - that helps me with phpstan and... in future.